db/migrate/20230203133927_add_recommendation_note_to_assessment.rb

```diff
@@ -0,0 +1,5 @@
+class AddRecommendationNoteToAssessment < ActiveRecord::Migration[7.0]
+  def change
+    add_column :assessments, :recommendation_assessor_note, :text
```

Comment:
```suggestion
    add_column :assessments, :recommendation_assessor_note, :text, null: false, default: ""
```

At the moment the field can either be `""` or it can be `nil` and they can both represent the absence of the note. I like to require that strings are not nullable but default to a blank string so there's only one possible value which represents the absence of a note.